### PR TITLE
added getters for AZSL surface params

### PR DESCRIPTION
**Pull request by antonmic**
added getters for surface normals to differentiate between diffuse normal and specular normal. Also added getter for specular f0

### CODE REVIEW

File: Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/StandardSurface.azsli [hunk 1]
```diff
@@ -78,6 +78,12 @@ class Surface
 
     //! Sets albedo, base color, specularF0, and metallic properties using metallic workflow
     void SetAlbedoAndSpecularF0(real3 baseColor, real specularF0Factor, real metallic);
+
+    real3 GetDiffuseNormal() { return normal; }
+    real3 GetSpecularNormal() { return normal; }
+    real3 GetDefaultNormal() { return normal; }
```
Comment (AMZN-alexpete) [inline Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/StandardSurface.azsli:84]: I'm not super clear on the use of DefaultNormal, some comments might be helpful here.

**Review by AMZN-alexpete — APPROVED** [review]
I'm OK with the change but am unclear on what "DefaultNormal" is for, and think that could use some comments.

File: Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/StandardSurface.azsli (hunk 1 above)
Comment (gadams3) [reply Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/StandardSurface.azsli:84]: +1 Default in what context? Is there a more specific name that can be used here?

**Review by gadams3 — APPROVED** [review]

**Review by galibzon — APPROVED** [review]

File: Gems/Atom/Feature/Common/Assets/Shaders/Materials/Skin/Skin_SurfaceData.azsli [hunk 2]
```diff
@@ -30,7 +30,7 @@ class SurfaceData_Skin
     real3 normal;              //!< Normal in world-space
     real3 vertexNormal;        //!< Vertex normal in world-space
     real3 albedo;              //!< Albedo color of the non-metallic material, will be multiplied against the diffuse lighting value
-    real3 specularF0;          //!< Fresnel f0 spectral value of the surface
+    real specularF0;          //!< Fresnel f0 spectral value of the surface
```
Comment (moudgils) [inline Gems/Atom/Feature/Common/Assets/Shaders/Materials/Skin/Skin_SurfaceData.azsli:33]: Expand the comment about why this is not float3.

**Pull request merged**